client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/package-info.js

```diff
@@ -159,11 +158,28 @@ const PackageInfo = props => {
 			props.addPackage( siteId );
 		};
 
+		const totalPackagesCount = reduce( packageGroups, ( acc, cur ) => {
+			return acc + cur.definitions.length;
+		}, 0 );
+
 		return (
-			<>
-			<div>
+			! totalPackagesCount ?
+			( <div className="packages-step__no-packages">
+				{ /* TODO: Style it! */ }
```

Comment:
Is there an issue created for the TODO here?

Reply:
Whoops! I've been meaning to create an issue but forgot :) Thanks!

https://github.com/Automattic/woocommerce-services/issues/1687